packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2691,13 +2691,13 @@ module.exports = function (log, config, oauthdb) {
     links.cancellationSurveyLinkAttributes = `href="${links.cancellationSurveyUrl}" style="text-decoration: none; color: #0060DF;"`;
 
     links.subscriptionTermsUrl = this._generateUTMLink(
```

Comment:
Hmm I guess this is not really necessary anymore since the UTM params won't do anything on the download links.